src/handleDisplay.js

```diff
@@ -0,0 +1,9 @@
+import displayItems from './displayList';
```

Comment:
DisplayItems is same as display list please use the same name for easier code readability